src/Persistence/Models/Component.cs

```diff
@@ -39,11 +38,11 @@ public Component(string name, string variant, IControlTemplateStore controlTempl
     public bool AccessAppScope { get; set; }
 
     [YamlMember(Order = 100)]
-    public CustomPropertiesCollection CustomProperties { get; set; } = new();
+    public IList<CustomProperty> CustomProperties { get; set; } = new List<CustomProperty>();
```

Comment:
are we going to delete the `CustomPropertiesCollection` now?

Reply:
I think so because NamedObjectsCollectionConverter can handle any named objects collection which implement simple INamedObject interface